package.json

```diff
@@ -65,7 +65,7 @@
     "rc-pagination": "~1.12.4",
     "rc-progress": "~2.2.2",
     "rc-rate": "~2.3.0",
-    "rc-select": "~7.3.2",
+    "rc-select": "^7.5.0",
```

Comment:
Use `~` instead of `^`.

Reply:
still failing

Reply:
I updated the `feature-3.1` branch, try rebase your branch on it.